CribblyBackend/Controllers/TeamController.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using CribblyBackend.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CribblyBackend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class TeamController : ControllerBase
+    {
+        private readonly ITeamService teamService;
+        public TeamController(ITeamService teamService)
+        {
+            this.teamService = teamService;
+        }
+
+        /// <summary>
+        /// GetById fetches the specified team.
+        /// </summary>
+        /// <param name="id">The id of the team to get</param>
+        /// <returns></returns>
+        [HttpGet]
+        public async Task<IActionResult> GetById([FromQuery] int id)
```

Comment:
Do we want the id to come from a query string (i.e. `/api/team?id={id}`) or just from another part of the route (i.e. `/api/team/{id}`? I'd vote for the latter since it's simpler 🤷‍♂️

Reply:
I agree, and changed